snuba/subscriptions/worker.py

```diff
@@ -47,3 +50,24 @@ def flush_batch(self, batch: Sequence[Sequence[SubscriptionResult]]) -> None:
         for results in batch:
             for result in results:
                 self.__producer.produce(self.__topic, result).result()
+
+
+class SubscriptionResultCodec(Codec[KafkaPayload, SubscriptionResult]):
+    def encode(self, value: SubscriptionResult) -> KafkaPayload:
+        subscription_id = str(value.task.task.identifier)
+        return KafkaPayload(
+            str(subscription_id).encode("utf-8"),
+            json.dumps(
+                {
+                    "version": 1,
+                    "payload": {
+                        "subscription_id": str(subscription_id),
+                        "values": value.result["data"],
```

Comment:
I was thinking this would include the entire `ClickhouseQueryResult`.

Reply:
Ahh, didn't quite follow your meaning. That seems fine, we might be able to use it for debugging on our end at some point too.